crates/ast/src/parser.rs

```diff
@@ -841,6 +849,34 @@ mod tests {
         );
     }
 
+    #[test]
+    fn parse_sol_event_indexed() {
```

Comment:
Let's keep the current schema of one test function with multiple assets per parser function

Reply:
Done 👌